components/layout/index.scss

```diff
@@ -0,0 +1,81 @@
+$sidebarWidth: 10rem;
+
+.layout {
+  grid-template-rows: min-content auto;
+  position: relative;
+  min-height: 100vh;
```

Comment:
I would prefer percents for safety. For example, in Reader's case if you embed the Reader (with this property) into a modal window you will experience issues.